### PR TITLE
Renamed OI problems for consistency

### PR DESCRIPTION
**Pull request by dolphingarlic**
- File names are of the format "{Olympiad in uppercase} {Last two digits of year}-{Problem name joined by underscores}.mdx"
- Problem IDs are of the format "{Olympiad in lowercase}-{Last two digits of year}-{Problem name joined by underscores}"
- If the problem has a "code" (e.g. POI/BOI 2015/CEOI 2018) or a name in a different language (e.g. LMiO), then we use that as the name instead

I might've left stuff out, so please double-check

### CODE REVIEW

**Review by dongliu0426 — CHANGES_REQUESTED** [review]
- https://usaco-guide-klg3e1b8q.vercel.app/gold/stacks?lang=cpp#problem-https://codeforces.com/contest/1402/problem/A
- https://usaco-guide-klg3e1b8q.vercel.app/gold/stacks?lang=cpp#problem-https://oj.uz/problem/view/COI15_zarulje

Add years to the problem name

**Review by dongliu0426 — APPROVED** [review]
Looks good for the gold content (oops)

File: content/4_Gold/DP_Trees.mdx [hunk 1]
```diff
@@ -34,8 +34,8 @@ export const problems = {
 			'similar to independent set on tree'
 		),
 		new Problem(
-			'CF',
-			'Baltic OI - Village (Minimum)',
+			'Baltic OI',
+			'2020 - Village (Minimum)',
 			'contest/1387/problem/B1',
```
Comment (dolphingarlic) [inline content/4_Gold/DP_Trees.mdx:39]: Oops I should fix these links too

File: content/5_Plat/DC-DP.mdx [hunk 2]
```diff
@@ -5,17 +5,23 @@ author: Andi Qu, Benjamin Qi
...
-
 export const problems = {
 	dnc_sample: [
 		new Problem('Plat', 'Circular Barn', '626', 'Hard', false, [], ''),
 	],
 	dnc_general: [
-		new Problem('CEOI', 'Two Sawmills', 'https://szkopul.edu.pl/problemset/problem/ovRIpLFK3IhyFPjnVXeZtGxH/site/?key=statement', 'Normal', false, [], ''),
+		new Problem(
+			'CEOI',
+			'Two Sawmills',
```
Comment (dolphingarlic) [inline content/5_Plat/DC-DP.mdx:18]: And this one

File: content/6_Advanced/LCT.mdx [hunk 3]
```diff
@@ -84,8 +84,8 @@ export const problems = {
 			''
 		),
 		new Problem(
-			'CF',
-			'Baltic OI - Joker',
+			'Baltic OI',
+			'2020 - Joker',
 			'contest/1386/problem/C',
```
Comment (dolphingarlic) [inline content/6_Advanced/LCT.mdx:89]: And this link

File: content/6_Advanced/Offline_Del.mdx [hunk 4]
```diff
@@ -53,8 +53,8 @@ export const problems = {
 			'Editorial solution is online N sqrt N, but you can solve this in offline N log N using this trick.'
 		),
 		new Problem(
-			'CF',
-			'Baltic OI - Joker',
+			'Baltic OI',
+			'2020 - Joker',
 			'contest/1386/problem/C',
```
Comment (dolphingarlic) [inline content/6_Advanced/Offline_Del.mdx:58]: And this link

**Review by dolphingarlic — COMMENTED** [review]
Oops I still left out a few

**Comment by dolphingarlic** [comment]
@dongliu0426 should I merge?

**Review by dongliu0426 — CHANGES_REQUESTED** [review]
oops there's still some stuff left out
- Silver bipartite -- Baltic OI Graph
- Silver intro-tree -- POI Hotels
- Adv CntMin -- IOI Pyramid Base
- Adv Slope -- LMiO Potatoes & Fertilizers
- Adv BCC - 2CC -- POI Blockade
- Adv SCC -- POI Bike Paths
- Adv LCT -- IOI Dancing Elephants
- Adv comb-sub -- COCI Dzumbus

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**